backend/fms_core/viewsets/_fetch_data.py

```diff
@@ -362,7 +366,7 @@ def fetch_export_data(self, ids: List[int] =[]) -> Tuple[List, int]:
                 'pedigree': derived_sample["biosample__individual__pedigree"] if not is_pool or not is_library else None,
                 'quality_flag': ["Failed", "Passed"][sample["quality_flag"]] if sample["quality_flag"] is not None else None,
                 'quantity_flag': ["Failed", "Passed"][sample["quantity_flag"]] if sample["quantity_flag"] is not None else None,
-                'projects': derived_sample["project__name"] if not is_pool else None,
+                'project': projects[sample["first_project_id"]] if not is_pool else None,
```

Comment:
Ah. So this is where the singular project is coming from for the `SampleExportSerializer`. Right?